### DISCUSSION TITLE
Feature request: Sensors Remake

### DISCUSSION THREAD

**Discussion by milandzuris** [dougiteixeira/proxmoxve#121] [Feature request] [answered]
1. Change Uptime time to minutes after minutes to hours after hours to days
2. Disk Used in % (no free)
3. Swap Used in %
4. Memory RAM Used in %

#EXAMPLE done Sensors in HA
CPU Used %
CPU Free %

RAM Used %
RAM Free%
RAM Used GB
RAM Free GB
RAM Total GB

DISK Used %
DISK Free %
DISK Used GB
DISK Free GB
DISK Total GB

SWAP Used %
SWAP Free %
SWAP Used GB
SWAP Free GB
SWAP Total GB

Sorry but you have big mess in Sensors, change mainly this Used GB to %, THX.

**Comment by dougiteixeira** [comment, marked as answer]
> 1. Change Uptime time to minutes after minutes to hours after hours to days

The uptime sensor has the TIMESTAMP device class, the frontend takes care of formatting the time. Besides that using an uptime sensor that is increasing is harmful to the database.

**Comment by DeepCoreSystem** [comment]
Hi there!

As I have installed today this integration in HomeAssistant, I have to disagree with the answer given.

The sensor "Uptime" should be renamed to "Last boot", because this is what this sensor of type TIMESTAMP does show in fact.

As opposed to this sensor of some other external hardware integrated into HomeAssistant, which shows the elapsed seconds since the last restart of this sensor.
![grafik](https://github.com/dougiteixeira/proxmoxve/assets/3703938/cc3909f7-ec6c-40db-8fc7-e86aa2eebe88)
